src/mailtemplates/getting-started/getting-started.html.mustache

```diff
@@ -466,7 +466,7 @@
                                     <table border="0" cellpadding="0" cellspacing="0" width="100%" id="templateHeader">
                                         <tr>
                                             <td valign="top" class="headerContent">
-                                            	<img src="https://www.unleash-hosted.com/wp-content/uploads/2019/12/unleash-hosted-logo-v2-mini-fit.png" style="max-width:600px;" id="headerImage" mc:label="header_image" mc:edit="header_image" mc:allowdesigner mc:allowtext />
+                                            	<img src="https://www.getunleash.io/logos/unleash_pos.svg" style="max-width:600px;padding:1rem;" id="headerImage" mc:label="header_image" mc:edit="header_image" mc:allowdesigner mc:allowtext />
```

Comment:
this is probably ok (and better than the previous). But I would prefer this to be a static/immutable CDN  url. I am working in establishing cdn.getunleash.io, we can change this when that is ready.

Reply:
Perfect.